src/include/stir/numerics/BSplinesDetail.inl

```diff
@@ -17,6 +17,8 @@
 */
 #include "stir/assign.h"
 
+#ifndef BSPLINESDETAIL_H
+#define BSPLINESDETAIL_H
```

Comment:
should never be necessary in a .inl file, but only in the .h

Reply:
There isnt a `BSplinesDetails.h` and I was running into issues with recompiling it more than once.

Reply:
ah. well, that file should have been called .h then...  sorry. In that case, call the variable `__stir_numerics_BSplinesDetail_inl_` then to avoid (unlikely) conflicts

Reply:
It appears it is used twice:

```
$grep -rn "BSplinesDetail.inl" .
Binary file ./.git/index matches
./src/include/stir/numerics/BSplinesRegularGrid.inl:19:#include "stir/numerics/BSplinesDetail.inl"
./src/include/stir/numerics/BSplines1DRegularGrid.inl:19:#include "stir/numerics/BSplinesDetail.inl"
```

Reply:
yes, and it breaks my rule that a .inl should not be included anywhere else than in its .h file. So you could rename the file, or just stick in the #ifdef. Probably best to do the latter, as the TOF PR is already big enough...